src/compass/utils/h5_helpers.py

```diff
@@ -398,6 +398,20 @@ def metadata_to_h5group(parent_group, burst, cfg):
     # create metadata group to write datasets to
     meta_group = parent_group.require_group('metadata')
 
+    # RFI information - placeholder
+    rfi_items = [
+        Meta('is_rfi_info_available', True,
+             'Whether RFI information is available'),
```

Comment:
```suggestion
             'Boolean variable specifying availability of RFI information'),
```
Despite I do not know how this metadata field is different from the one below. Maybe historical data do not have RFI info?

Reply:
> Maybe historical data do not have RFI info?
Yes, the RFI information is available from IPF version 3.40. `is_rfi_info_available` is to explicitly indicate that the information is not available.
`rfi_mitigation_performed` is whether RFI mitigation was performed. The existence of this information means that the RFI information is available.